cuvis/SessionFile.py

```diff
@@ -25,11 +25,24 @@ def __init__(self, base: Union[Path,str]):
 
     pass
 
-    def get_measurement(self, frameNo: int, itemtype: SessionItemType = SessionItemType.no_gaps) -> Measurement:
+    def get_measurement(self, frameNo: int, itemtype: SessionItemType = SessionItemType.no_gaps) ->  Optional[Measurement]:
         _ptr = cuvis_il.new_p_int()
-        if cuvis_il.status_ok != cuvis_il.cuvis_session_file_get_mesu(
-                self._handle, frameNo, internal.__CuvisSessionItemType__[itemtype],
-                _ptr):
+        ret =  cuvis_il.cuvis_session_file_get_mesu(self._handle, frameNo, internal.__CuvisSessionItemType__[itemtype],
+                _ptr)
+        if cuvis_il.status_no_measurement == ret:
+            return None
+        if cuvis_il.status_ok != ret:
+            raise SDKException()
+        return Measurement(cuvis_il.p_int_value(_ptr))
+    
+    def get_reference(self, frameNo: int, reftype: ReferenceType) ->  Optional[Measurement]:
+        _ptr = cuvis_il.new_p_int()
+        ret = cuvis_il.cuvis_session_file_get_reference_mesu(
+                self._handle, frameNo, internal.__CuvisSessionItemType__[reftype],
+                _ptr)
+        if cuvis_il.status_no_measurement == ret:
```

Comment:
How does C/Cpp SDK report request that ends in  "no_measurement?" Is there an info that nothing was found? 

otherwise, include `print("reference {} not found".format(reftype)`

Reply:
well, it does not find the reference that gets requested by the user